finetune_instruct_pix2pix.py

```diff
@@ -1075,7 +1081,8 @@ def collate_fn(examples):
                 pipeline.set_progress_bar_config(disable=True)
 
                 # run inference
-                original_image = download_image(args.val_image_url)
+                # original_image = download_image(args.val_image_url)
+                original_image = open_image(args.val_image_url)
```

Comment:
This won't work if you pass an image URL. It will only work if you pass an image file path that is local in nature.

Reply:
Pushing a change for this one to accept either a URL or a local file path